.travis.yml

```diff
@@ -55,6 +55,15 @@ cache:
     - $HOME/gcloud/
     - $HOME/site-library
 before_install:
+  #skip push builds if there's an associated PR because we don't look at them anway
+- if [[ ${TRAVIS_EVENT_TYPE} == push ]];
+  then
+    PULL_REQUESTS=$(curl -v -H "Authorization':' token $GITHUB_API_TOKEN" https://api.github.com/repos/broadinstitute/gatk/pulls\?state\=open\&head="broadinstitute:${TRAVIS_BRANCH}");
+    if [[ $( grep -c "commits" <<< ${PULL_REQUESTS} ) -gt 0 ]];
```

Comment:
Instead of grepping for a specific term, perhaps it would be better to check that the result is non-empty after trimming `[     ]`. Eg., check that:

```
curl https://api.github.com/repos/broadinstitute/gatk/pulls\?state\=open\&head="broadinstitute:lb_skip_push_buildsfoo" | sed 's/\[//g' | sed 's/\]//g' | tr -d '[:space:]' | wc -c
```

is non-zero.

Reply:
I'm going to skip this because I'm not convinced it's any more robust to changes in the returned format.